packages/wonder-blocks-tooltip/src/components/__tests__/tooltip.test.js

```diff
@@ -32,234 +32,144 @@ describe("Tooltip", () => {
-        test("with title, sets title of TooltipContent", async () => {
-            // Arrange
-            const ref = await new Promise((resolve) => {
-                const content = (
-                    <TooltipContent>
-                        <HeadingSmall>Some custom content</HeadingSmall>
-                    </TooltipContent>
-                );
-                const title = <HeadingSmall>Title</HeadingSmall>;
-                const nodes = (
-                    <View>
-                        <Tooltip id="tooltip" title={title} content={content}>
-                            <View ref={resolve}>Anchor</View>
-                        </Tooltip>
-                    </View>
-                );
-                mount(nodes);
-            });
-            const node = (ReactDOM.findDOMNode(ref): any);
-            node && node.dispatchEvent(new KeyboardEvent("focusin"));
+            const node = screen.getByText("Anchor");
+            userEvent.hover(node);
             jest.runOnlyPendingTimers();
 
             // Act
-            // Flow doesn't like jest mocks
-            // $FlowFixMe[prop-missing]
-            const result = TooltipBubble.mock.instances[0].props["children"];
+            const tooltip = screen.getByRole("tooltip");
 
             // Assert
-            expect(result.props["title"]).toMatchSnapshot(
-                `Similar to <HeadingSmall>Title</HeadingSmall>`,
-            );
-            expect(result.props["children"]).toMatchSnapshot(
-                `Similar to <HeadingSmall>Some custom content</HeadingSmall>`,
-            );
+            expect(tooltip).toBeInTheDocument();
         });
 
-        test("with title, overrides title of TooltipContent", async () => {
+        it("should show the tooltip on focusin", async () => {
             // Arrange
-            const ref = await new Promise((resolve) => {
-                const content = (
-                    <TooltipContent title="Content title">
-                        <Body>Some custom content</Body>
-                    </TooltipContent>
-                );
-                const nodes = (
-                    <View>
-                        <Tooltip id="tooltip" title="Title" content={content}>
-                            <View ref={resolve}>Anchor</View>
-                        </Tooltip>
-                    </View>
-                );
-                mount(nodes);
-            });
-            const node = (ReactDOM.findDOMNode(ref): any);
-            node && node.dispatchEvent(new KeyboardEvent("focusin"));
+            render(
+                <View>
+                    <Tooltip title="Title" content="Content">
+                        <View>Anchor</View>
+                    </Tooltip>
+                </View>,
+            );
+
+            const node = screen.getByText("Anchor");
+            userEvent.hover(node);
+            jest.runOnlyPendingTimers();
+            userEvent.unhover(node);
```

Comment:
We've been talking about this in Perseus. In this test, do you think we should assert that the tooltip shows before asserting that it disappears? There's been conversation about whether you should have that assertion in the "disappears" test or whether it's enough that the preceding test is asserting that the thing shows.

I worry about false positives (ie "it's not in the document, not because it left the DOM, but because it never appeared"), but obviously the previous test would catch this.